modules/deprecated-feature.adoc

```diff
@@ -0,0 +1,11 @@
+// When including this file, ensure that {FeatureName} is set immediately before
+// the include. Otherwise it will result in an incorrect replacement.
+
+[IMPORTANT]
+====
+{FeatureName} is a deprecated feature. Deprecated functionality is still included in OpenShift Container Platform and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.
+
+For the most recent list of major functionality deprecated and removed within {product-title}, refer to the _Deprecated and removed features_ section of the {product-title} release notes.
```

Comment:
Suggest clarifying the wording slightly so that deprecated and removed don't sound as paired:
s/functionality deprecated and removed/functionality that has been deprecated or removed/